src/errors.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Errors related to the BlindBid module
+
+use dusk_jubjub::JubJubScalar;
+#[cfg(feature = "std")]
+use std::fmt;
+
+#[derive(Debug)]
+/// Definition of the erros that Bid operations might have.
+pub enum BlindBidError {
+    /// Error for the cases when we the score results are too large to
+    /// fit inside a `Scalar`.
+    InvalidScoreFieldsLen,
+    /// Error that happens when you try to generate a `Score` for a `Bid`
+    /// has already expired.
+    ExpiredBid,
+    /// Error for the cases when we the provided Bid value is bigger
+    /// than the maximum allowed by the specs..
+    MaximumBidValueExceeded {
+        /// The maximum bid_value allowed
+        max_val: JubJubScalar,
+        /// The expected length
+        found: JubJubScalar,
+    },
+    /// Error for the cases when we the provided Bid value is lower
+    /// than the minimum allowed by the specs..
+    MinimumBidValueUnreached {
+        /// The minimum bid_value required
+        min_val: JubJubScalar,
+        /// The expected length
+        found: JubJubScalar,
+    },
+    /// Error when there is a decrypt attempt with the wrong secret
+    WrongSecretProvided,
+    /// Invalid encoding/decoding
+    IOError,
```

Comment:
Do we still have I/O anywhere?

Reply:
It's for the to_bytes/from_bytes methods.